sources/HUBUtilities.h

```diff
@@ -24,10 +24,31 @@
 #import "HUBComponent.h"
 #import "HUBJSONCompatibleBuilder.h"
 #import "HUBSerializable.h"
+#import "HUBKeyPath.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 /**
+ *  Return whether two properties are considered equal
+ *
+ *  @param objectA The first object
+ *  @param objectB The second object
+ *  @param propertyName The name of the property to compare
+ *
+ *  Two nil values are considered equal.
+ */
+static inline BOOL HUBPropertyIsEqual(NSObject * _Nullable objectA, NSObject * _Nullable objectB, NSString *propertyName) {
```

Comment:
No reason, just in the habit of doing it. Adding `inline` as no downsides AFAIK though 😄